Composer/packages/server/src/utility/storage.ts

```diff
@@ -29,8 +29,7 @@ export async function copyDir(srcDir: string, srcStorage: IFileStorage, dstDir:
 
     if ((await srcStorage.stat(srcPath)).isFile) {
       // copy files
-      const content = await srcStorage.readFile(srcPath);
-      await dstStorage.writeFile(dstPath, content);
+      await srcStorage.copyFile(srcPath, dstPath);
```

Comment:
Sorry, this is not right, because this read and write mechanism can handle there is different storage for src and dst. But in this change, it's copying from src to src. 

Can we do a condition to check if src is same as dst, then do the copyFile, otherwise we still do read\write?

Reply:
> Sorry, this is not right, because this read and write mechanism can handle there is different storage for src and dst. But in this change, it's copying from src to src.
> 
> Can we do a condition to check if src is same as dst, then do the copyFile, otherwise we still do read\write?

ok I will fix this